.travis.yml

```diff
@@ -31,9 +31,8 @@ before_script:
 script:
   - go build -v ./cmd/dep
   - go vet $PKGS
-  # TODO (kris-nova) remove the ignore flag once we can refactor the code causing problems
```

Comment:
Technically didn't mean to remove this here, rebase conflict with #554, should be harmless.